client/src/components/UnsavedChangesIndicator/UnsavedChangesIndicator.scss

```diff
@@ -0,0 +1,70 @@
+.unsaved-changes-indicator__container {
+  display: flex;
+}
```

Comment:
Is this only used in the entwine? If so, why?

Reply:
Without this the vertical alignment on the indicator will be wrong

Reply:
Why is it only used in entwine, though? Why isn't this as a parent element just part of the react component?

Reply:
It's standard to have a `__container` class in entwine defined in PHP or the template to act as a target for the react component. The react component does not replace the target.

e.g. https://github.com/silverstripe/silverstripe-versioned-admin/blob/3/client/src/legacy/HistoryViewer/HistoryViewerEntwine.js#L11

> Why isn't this as a parent element just part of the react component?

In the entwine context, The `display: flex` needs to be on the element that's a direct child of `btn-toolbar` in order for the vertical alignment to work right.  In this case the direct child has to be the entwine target, not the react component.

Reply:
I've added a large inline comment explaining this